src/dynamodb/client.py

```diff
@@ -236,3 +263,6 @@ def bulk_insert_github_handle_to_asana_user_id_mapping(
     DynamoDbClient.singleton().bulk_insert_github_handle_to_asana_user_id_mapping(
         gh_and_asana_ids
     )
+
+def _get_dynamodb_key_from_two_github_nodes(gh_node_id_a: str, gh_node_id_b: str) -> str:
+    return f"{gh_node_id_a}-{gh_node_id_b}"
```

Comment:
Perhaps this function should sort the two node ids, so we don't have to remember which order to pass them in and reduce bugs around unexpected orders